src/server/utils/Errors.ts

```diff
@@ -29,7 +31,9 @@ export const ErrorMessage = {
   EXPECTED_QUERY_PARAM: param => `Expected query parameter not provided '${param}'`,
   INVALID_RESET: () => 'This reset request is invalid or has expired',
   USER_HAS_NO_TEAM: () => 'This user is not part of a team',
-  TEAM_FULL: (teamCode, maxSize) => `'${teamCode}' already has the maximum allowed ${maxSize} members`,
+  USER_ON_TEAM: () => `A user is already a part of the team`,
```

Comment:
Can you make this part of `a` team? Assume we have 2 teams with one user each and i try to add one user to the other team, this text implies that user 2 is a part of team 1 while what we really mean is that they are associated with _another_ team. 

![image](https://user-images.githubusercontent.com/23143381/102743265-1c257580-4325-11eb-9d30-8c108de03ac0.png)
![image](https://user-images.githubusercontent.com/23143381/102743282-25aedd80-4325-11eb-91ff-418b5b9bf14c.png)

Reply:
+1 "a team" or "another team" would be clearer